vortexasdk/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.68.2"
+__version__ = "0.68.3"
```

Comment:
Nitpick(optional): this should be a minor bump rather than a patch.

(I'm not too fussed though if this typo was shipped very recently.)